src/adapters/poi_popup.js

```diff
@@ -42,39 +46,49 @@ PoiPopup.prototype.addListener = function(layer) {
   })
 }
 
-PoiPopup.prototype.create = async function (layerPoi, event) {
+PoiPopup.prototype.createOSMPopup = async function (layerPoi, event) {
   let poi = await ApiPoi.poiApiLoad(layerPoi.properties.global_id, {simple : true})
   if(poi) {
-    if(this.popupHandle) {
-      this.popupHandle.remove()
-      this.popupHandle = null
-    }
-    let {color} = IconManager.get(poi)
-    let category = poiSubClass(poi.subClassName)
-    let hours = poi.blocks.find(block =>
+    this.showPopup(poi, event);
```

Comment:
Sorry, It looks like we do not use semicolons in this project. It is pretty unpleasing to make the devs go back to their PRs just for those kind of things. Maybe we could introduce a code formatter as a precommit hook in the future or eslint

Reply:
;)